ovos_tts_plugin_server/__init__.py

```diff
@@ -57,11 +57,9 @@ def get_tts(
         if not voice or voice == "default":
             params.pop("voice")
         if self.host:
-            if isinstance(self.host, str):
-                servers: List[str] = [self.host]
-            else:
-                servers = self.host
+            servers = self.host
         else:
+            random.shuffle(self.public_servers)
```

Comment:
You will never hit this condition because `self.host` always returns a truthy value, e.g. a list of strings

Reply:
This is working perfectly on my system, because when `self.config.get` fails, it returns None, not a list

Reply:
After reviewing the thread above, that makes sense, but it should never return None - it would be a list of None, which also evaluates as falsey